src/test/java/POM/BasePage.java

```diff
@@ -33,16 +34,5 @@ public boolean isUserAvatarDisplayed(){
         return driver.findElement(avatarLocator).isDisplayed();
     }
 
-    public boolean isSongPlaying() {
-        WebElement soundBarVisualizer = driver.findElement(soundBarPlayLocator);
-        return soundBarVisualizer.isDisplayed();
-    }
 
-    public AllSongsPage clickOnAllSongs(){
-        driver.findElement(allSongsMenuItemLocator).click();
-        return new AllSongsPage(driver);
-    }
-    public void chooseAllSongsList() {
-        wait.until(ExpectedConditions.visibilityOfElementLocated(By.cssSelector("li a.songs"))).click();
-    }
```

Comment:
1.**I recommend: make the variables protected,** because protected access gives the subclass a chance to use the helper method or variable, while preventing a nonrelated class from trying to use it.

For example

```
  protected WebDriver driver;
  protected WebDriverWait wait;
  protected Actions actions;
```
2.Base Page \ Base Class — Implements the necessary methods for work with webdriver.

We will store the web elements present on each page in the corresponding java class.

The methods `getUserAvatar ()`,isUserAvatarDisplayed() must be stored in the `class HomePage` because the avatar is visible on the web site -home page

![image](https://user-images.githubusercontent.com/104761482/207937606-83ca466c-bec3-4324-bfa9-6678662ac92e.png)